arches/app/media/js/views/components/widgets/urldatatype.js

```diff
@@ -11,6 +11,16 @@ define([
          
         WidgetViewModel.apply(this, [params]);
 
+
+        // #10027 assign this.url & this.url_label with value versions for updating UI with edits
```

Comment:
Noticed that with this change, if I save a tile with a URL and without a label, the dirty state is still active, i.e. "cancel edit" and "save" buttons still present.

This is tricky because the old behavior is definitely more buggy, but would be nice to leave the situation improved in all respects.

Reply:
I'm not sure I can reproduce this, once saving the URL tile (with no label) the edit and save buttons are gone for me. 
https://github.com/archesproject/arches/assets/119508494/50376937-7c41-44e8-bbf5-df1fe7a7c40f

Am I trying the wrong thing, or could it be something wrong with my testing environment?

Reply:
I cannot reproduce this problem any longer. Thanks for bearing with me.